### PR TITLE
CEF-3982: Aligned systemd service templates with core

### PR DESCRIPTION
**Pull request by nickanderson**

### CODE REVIEW

File: templates/cf-apache.service.mustache [hunk 1]
```diff
@@ -17,4 +17,3 @@ UMask=0177
 
 [Install]
 WantedBy=multi-user.target
-WantedBy=cfengine3.service
```
Comment (olehermanse) [inline templates/cf-apache.service.mustache:20]: Could put link to related PR / commit in commit message.

Changelog entry does not explain the result of this change to a user.

**Review by olehermanse — CHANGES_REQUESTED** [review]

File: templates/cf-apache.service.mustache (hunk 1 above)
Comment (nickanderson) [reply templates/cf-apache.service.mustache:20]: changed.

**Review by olehermanse — APPROVED** [review]
LGTM

**Review by vpodzime — APPROVED** [review]
ACK

**Pull request merged**